examples/tracer-web/README.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+This example shows how to use [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) to instrument your JavScript code running in the browser.
+The supplied example uses the `ConsoleSpanExporter()` to export spans into the console output.
+
+## Installation
+
+```sh
+$ # from this directory
+$ npm install
+```
+
+## Run the Application
+
+```sh
+$ # from this directory
+$ npm start
+```
+
+By default, the application will run on port `8090`: `http://localhost:8090/`
```

Comment:
Maybe mention that you have to actually open the application in a web browser to generate a trace?

Reply:
Done.